README.md

```diff
@@ -445,10 +445,9 @@ command:
 make BOARD=linux
 ```
 
-The executable will be outputted to `outdir/linux/<variant>/jslinux`. Where
-`<variant>` is either `debug` or `release`. This is specified the same as the
-Zephyr target by passing in `VARIANT=` when running `make`. The default is
-release.
+The executable will be outputted to `outdir/linux/jslinux`.  If `variant` is
+set to `debug`. The debug executable will be stored in `outdir/linux/debug/jslinux`.
```

Comment:
This 'If' sentence is incomplete and hence makes no sense...

Reply:
Why only have one build? That's what we do for other boards, but it's kind of nice to have both and switch between them when debugging something.

Reply:
they do build and the output is in different directories, and they don't conflict, so you could switch between them. The release sits just 1 directory higher now, so it's outdir/linux/jslinux instead of outdir/linux/release/jslinux